### ISSUE TITLE
Issues with new Gulp file

### ISSUE THREAD

**Issue by Sally-Xu** [wsick/Fayde#171]
Just updated the latest Fayde project, Looks like you switched from Grunt to Gulp now. Somehow running Gulp reset or symlink only creates empty lib structure under the testsite, test and stress, but no files under those folders. 

Looks like Gulp symlink has a problem, here is the error message:

cmd.exe /c gulp -b "D:\Project\FaydeProject\Fayde" --color --gulpfile "D:\Project\FaydeProject\Fayde\Gulpfile.js" symlink-testsite
[11:58:54] Using gulpfile D:\Project\FaydeProject\Fayde\Gulpfile.js
[11:58:54] Starting 'symlink-testsite'...
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\lib\minerva symlinked to testsite\lib\minerva
[11:58:54] 'symlink-testsite' errored after 44 ms
[11:58:54] Error in plugin 'gulp-symlink'
Message:
    EEXIST, symlink 'D:\Project\lib\minerva' 
Details:
    errno: 47
    code: EEXIST
    path: D:\Project\lib\minerva
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\lib\nullstone symlinked to testsite\lib\nullstone
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\lib\perfex symlinked to testsite\lib\perfex
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\lib\requirejs symlinked to testsite\lib\requirejs
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\lib\requirejs-text symlinked to testsite\lib\requirejs-text
[11:58:54] gulp-symlink:D:\Project\FaydeProject\Fayde\dist symlinked to testsite\lib\fayde\dist

Looks like it is not looking for the correct lib path.

**Comment by BSick7** [comment]
Are you using visual studio to run?

Try running `gulp reset` from the command line. Of that works, then we can narrow in on the problem.

**Comment by Sally-Xu** [comment]
yes, I'm using visual studio's task runner.

**Comment by Sally-Xu** [comment]
Tried to run gulp reset from command line, still not working. I can see testsite/lib/minerva folder, but when I cd to this folder, then run "dir" command, I got this:

dir : Could not find a part of the path 'D:\Project\FaydeProject\Fayde\testsite\lib\minerva'.
At line:1 char:1
+ dir
+ ~~~
    + CategoryInfo          : ReadError: (D:\Project\Fayd...ite\lib\minerva:String) [Get-ChildI
   Exception
    + FullyQualifiedErrorId : DirIOError,Microsoft.PowerShell.Commands.GetChildItemCommand

When I run "Dir" under testsite/lib folder, I can see lib/fayde is a directory. but other folders are created as "Junction", they are pointing to  D:\Project\lib\XXX  which doesn't exist.

![screenshot](https://cloud.githubusercontent.com/assets/1042478/9155930/1e3bd5d2-3e97-11e5-8f29-7e32cc4ea474.png)

**Comment by BSick7** [comment]
Please try with latest.

**Comment by Sally-Xu** [comment]
Still not working. Still pointing to D:\Project\lib\XXX.

**Comment by BSick7** [comment]
Can you run `node -v` and `npm -v`?

**Comment by Sally-Xu** [comment]
node -v : v0.10.28
npm -v: 1.4.9

**Comment by Sally-Xu** [comment]
Just updated node to v0.12.7 and npm to v.2.11.3.

still not working.

**Comment by BSick7** [comment]
I was able to reproduce it.  Are you using win10?

**Comment by BSick7** [comment]
Found the issue to this. `gulp-symlink` uses symlinks as a first attempt.  It falls back to using junctions which do not support relative symlinks.  Since Fayde is using relative symlinks, it's intepreting the link incorrectly.

**Comment by BSick7** [comment]
As a workaround, run this with admin privileges.  The symlink attempt is failing due to a permissions issue.

**Comment by Sally-Xu** [comment]
yup, that worked. I usually always set to open visual studio as admin. I just forgot to do that after installing vs 2015.
